### PR TITLE
Query View page design adjustments

### PR DESCRIPTION
**Pull request by gabrieldutra**
## What type of PR is this? (check all applicable)

- [x] Other

## Description
Addresses comments from #4663 

[Will write a detailed explanation of what I did while netlify runs]

## Related Tickets & Documents
#4663 

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)
Soon

### CODE REVIEW

File: client/app/assets/less/inc/base.less [hunk 1]
```diff
@@ -244,18 +244,14 @@ text.slicetext {
   display: flex;
   align-items: center;
 
-  h3 {
-    margin-right: 5px !important;
```
Comment (gabrieldutra) [inline client/app/assets/less/inc/base.less]: This was more a DashboardPage thing, so moved to it.

File: client/app/assets/less/inc/base.less [hunk 2]
```diff
@@ -244,18 +244,14 @@ text.slicetext {
...
-
   .label {
     margin-top: 3px;
     display: inline-block;
   }
 
   .favorites-control {
     font-size: 19px;
-    margin-right: 5px;
+    margin-right: 10px;
```
Comment (gabrieldutra) [inline client/app/assets/less/inc/base.less]: This also affects the DashboardPage now, we added those extra 5px on the Query Pages now, I thought it should also be in there for consistence, LMK if you think it differently.

**Comment by arikfr** [comment]
One issue I noticed on mobile: charts have some min height which is probably too small:

Screen Shot 2020-02-24 at 11 33 21 AM

Another example:

image

Although to be honest, I'm not sure what the right solution here. Can min-height be calculated?

**Comment by gabrieldutra** [comment]
@arikfr I think it can be an unrelated bug with Chart as it seems it happened in Angular too:
chart-height-bug
(Showing both React and Angular versions)

Perhaps it's calculating its size too early? (notice that slightly changing the width makes it better fit the space)

@kravets-levko, do you know ath related?

**Comment by kravets-levko** [comment]
I didn't see this issue before. Looks like it's calculating it's size too early, but I have no idea why :thinking:

**Pull request merged**